### DISCUSSION TITLE
Can't use normal usb mouse scroll in Better touch tool while using MacMousefix running on Ventura?

### DISCUSSION THREAD

**Discussion by khurshid-alam** [noah-nuebling/mac-mouse-fix#992] [Questions]
I use MacMouseFix (2.4) only for smooth scrolling. No button related action. And while it is active can't use any trigger in Better touch tool with mouse scroll anywhere.

![](https://community.folivora.ai/uploads/default/original/3X/7/5/75f60c70f68dcdf484396768e613832535a671dc.jpeg)

Same thing SmoothScroll. Disabling those, everything works again.

In BTT settings I have "High level mouse event recognition" enabled as pointed out on reddit. But it **didn't** help.

![](https://community.folivora.ai/uploads/default/original/3X/c/5/c500c46c7d837cf54aafbf6b6a3363257a458aec.png)

However, there is an automation app [1Piece](https://app1piece.com) app which has this interesting settings for mouse wheel. Like scroll up on window titlebar to maximize window (One of my long time wish list for BTT ) and scroll down to restore. And guess what, scroll works as usual while both BTT and Macmousefix running! It also works with BTT. No conflict whatsoever!

![](https://community.folivora.ai/uploads/default/optimized/3X/5/9/599acacb7bb2264686edf3677940fe310baa9b64_2_1034x362.png)
)

I tried other applications like Steermouse with them, all at there same time, and it yet scrolling still works in 1Piece.

What could be the issue here ? Is this os/system related ? Obviously 1Piece does something that enables it to work Mac mouse fix or with any such app.

**Comment by TylerBarnes** [comment]
I noticed the same thing where none of those work, but https://www.smoothscroll.net/mac/ does work. However it causes other weird issues for me. Seems there's no perfect solution as MacMouseFix is my favourite of these apps, but it doesn't work with BTT

**Reply by TylerBarnes** [reply]
My specific issue is MacMouseFix prevents BTT from identifying when a modifier is held while scrolling. Seems like toggling off smooth scrolling makes it work - so it's definitely related to that.

**Reply by khurshid-alam** [reply]
Yes. But It should be working without holding modifier as well. Since I am not interested any button action other than smooth scroll. Linear mouse has this option to accommodate other input monitoring apps. Looks like we have to open a bug.


Btw, Mos released with new feature to control speed, acceleration and animation delay. After lots of trial and errors I reached this perfect settings which is just as good as macmouse fix. And it doesn’t block anything. It also has per app settings which working well for iterm. It seems I no longer need MMFix.

**Reply by TylerBarnes** [reply]
Nice! I gave it a shot and it works really well, my only issue is it interferes with a keyboard scrolling app I use (scrolla) but I guess you can't have everything 😄
